src/main/java/stringAddCalculator/StringAddCalculator.java

```diff
@@ -0,0 +1,29 @@
+package stringAddCalculator;
+
+import java.util.List;
+
+public class StringAddCalculator {
+    private static InputHandler inputHandler = new InputHandler();
+    private static Divider divider = new Divider();
+
+    public static int splitAndSum(String input) {
+        if (inputHandler.isEmptyInput(input)) return 0;
```

Comment:
```suggestion
        if (inputHandler.isEmptyInput(input)) {
            return 0;
        }
```
한줄로 쓰면 협업하는 개발자는 리턴값을 놓치기 쉽습니다.